diff --git a/filters/include/pcl/filters/voxel_grid_covariance.h b/filters/include/pcl/filters/voxel_grid_covariance.h
index b5624f1..a7909ee 100644
--- a/filters/include/pcl/filters/voxel_grid_covariance.h
+++ b/filters/include/pcl/filters/voxel_grid_covariance.h
@@ -82,6 +82,7 @@ namespace pcl
       using PointCloudConstPtr = typename PointCloud::ConstPtr;
 
     public:
+      PCL_MAKE_ALIGNED_OPERATOR_NEW
 
       using Ptr = shared_ptr<VoxelGrid<PointT> >;
       using ConstPtr = shared_ptr<const VoxelGrid<PointT> >;
