diff --git a/tools/pybind11Tools.cmake b/tools/pybind11Tools.cmake
index c255e5c..4f5ac72 100644
--- a/tools/pybind11Tools.cmake
+++ b/tools/pybind11Tools.cmake
@@ -119,8 +119,8 @@ set_property(
   TARGET pybind11::module
   APPEND
   PROPERTY
-    INTERFACE_LINK_LIBRARIES pybind11::python_link_helper
-    "$<$<OR:$<PLATFORM_ID:Windows>,$<PLATFORM_ID:Cygwin>>:$<BUILD_INTERFACE:${PYTHON_LIBRARIES}>>")
+    INTERFACE_LINK_LIBRARIES pybind11::python_link_helper)
+target_link_libraries(pybind11::module INTERFACE ${PYTHON_LIBRARIES})
 
 if(PYTHON_VERSION VERSION_LESS 3)
   set_property(
@@ -132,7 +132,8 @@ endif()
 set_property(
   TARGET pybind11::embed
   APPEND
-  PROPERTY INTERFACE_LINK_LIBRARIES pybind11::pybind11 $<BUILD_INTERFACE:${PYTHON_LIBRARIES}>)
+  PROPERTY INTERFACE_LINK_LIBRARIES pybind11::pybind11)
+target_link_libraries(pybind11::embed INTERFACE ${PYTHON_LIBRARIES})
 
 function(pybind11_extension name)
   # The prefix and extension are provided by FindPythonLibsNew.cmake
